### PR TITLE
enable overlays for minimal sizing

### PR DESCRIPTION
**Pull request by NormJohnIV**

### CODE REVIEW

**Comment by hahewlet** [comment]
@NormJohnIV You'd mentioned to me today that if the customer passes in the terraform.tfstate file, the value of V4_CFG_CLUSTER_NODE_POOL_MODE will be parsed from the integration. I don't see a doc change in README.md to reflect this.

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -228,6 +228,7 @@ When setting V4_CFG_TLS_MODE to a value other than "disabled" and no V4_CFG_TLS_
 
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
+| V4_CFG_CLUSTER_NODE_POOL_MODE | What mode of cluster node pool to use | string | "standard" | false | [standard, minimal] | viya |
```
Comment (hahewlet) [inline docs/CONFIG-VARS.md:231]: something is wrong with the editing change here. Look at it in rich diff. Many elastic search lines are listed as crossed out.

Comment (NormJohnIV) [reply docs/CONFIG-VARS.md:231]: I took a look at the page and don't see any incorrect formatting. https://github.com/sassoftware/viya4-deployment/blob/minimal_deploy/docs/CONFIG-VARS.md

**Comment by NormJohnIV** [comment]
> @NormJohnIV You'd mentioned to me today that if the customer passes in the terraform.tfstate file, the value of V4_CFG_CLUSTER_NODE_POOL_MODE will be parsed from the integration. I don't see a doc change in README.md to reflect this.

Updated: https://github.com/sassoftware/viya4-deployment/pull/111/commits/3f1648101263ae026d0963422e7ac2150ab260a2

File: docs/CONFIG-VARS.md (hunk 1 above)
Comment (hahewlet) [reply docs/CONFIG-VARS.md:231]: Yes that link looks good to me too. It's when I choose rich diff that things are not rendering correctly. I use rich diff all the time and it always renders correctly. When I've seen issue like the this in the past it was a syntax error. 

image

**Comment by hahewlet** [comment]
Testing is completed. Signing off.

**Pull request merged**